### ISSUE TITLE
Comprehensive example tasks for running multislice TPU workloads with JobSet (and JobSet + Kueue)

### ISSUE THREAD

**Issue by danielvegamyhre** [kubernetes-sigs/jobset#428]
**What would you like to be added**:
We added these comprehensive guides to Google Cloud documentation, so we should improve discoverability of them by adding them as tasks in the main repo. 
- https://cloud.google.com/kubernetes-engine/docs/how-to/tpu-multislice
- https://cloud.google.com/kubernetes-engine/docs/tutorials/tpu-multislice-kueue

**Why is this needed**:
We need more comprehensive examples that will reduce friction for users trying out JobSet for real training workloads. Right now we mostly just have toy examples with "sleep" containers that demonstrate different features.

**Comment by kannon92** [comment]
@0xlen do you have access to TPUs to do this?

cc @GiuseppeTT @imreddy13 

idk if this is really a priority anymore.

**Comment by GiuseppeTT** [comment]
I believe this fits in the general request of improving the JobSet documentation. It's a valid request and I want it to be done but I cannot prioritize this now.

In the meantime: 
- Kueue has an example on how to use Kueue with JobSet: https://kueue.sigs.k8s.io/docs/tasks/run/jobsets/
- GCP has an example on how to use JobSet for a multi slice workload: https://docs.cloud.google.com/kubernetes-engine/docs/how-to/tpu-multislice

**Comment by 0xlen** [comment]
Yes @kannon92, I can work on this, and thanks for the pointers @GiuseppeTT.

I plan to address this by adding a dedicated page to the JobSet documentation. Right now is reviewing the current materials/code base, my current idea is having a new `examples/tpu-multislice` directory with the JAX manifest + A new documentation page under "Tasks".

I will submit a PR and request for review/feedback for this.
